views/partials/header.ejs

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Red Balloon</title>
+    <title >Red Balloon</title>
```

Comment:
Maybe erase the space.

Reply:
I think I did it, let me know if it doesn't work.